src/main/java/com/google/api/codegen/transformer/java/JavaTestTransformer.java

```diff
@@ -86,11 +94,41 @@ private ApiTestClassView createTestClassView(Interface service) {
     testClass.apiClassName = surfaceNamer.getApiWrapperClassName(service);
     testClass.name = testClass.apiClassName + "Test";
     testClass.mockServiceClassName = "Mock" + testClass.apiClassName;
-    testClass.testCases = new ArrayList<>();
+    testClass.testCases = createTestCaseViews(service);
 
     String outputPath = pathMapper.getOutputPath(service, apiConfig);
     testClass.outputPath = outputPath + "/" + testClass.name + ".java";
     testClass.imports = typeTable.getImports();
     return testClass;
   }
+
+  private List<ApiTestCaseView> createTestCaseViews(Interface service) {
+    ArrayList<ApiTestCaseView> testCaseViews = new ArrayList<>();
+    for (Method method : service.getMethods()) {
+      MethodConfig methodConfig = apiConfig.getInterfaceConfig(service).getMethodConfig(method);
+      MethodTransformerContext context =
+          MethodTransformerContext.create(
+              service, apiConfig, typeTable, surfaceNamer, method, methodConfig);
+      ApiTestCaseView testCaseView = new ApiTestCaseView();
+      testCaseView.name = method.getSimpleName() + "Test";
```

Comment:
Add a method to a namer class (e.g. SurfaceNamer) to provide this for you. Maybe we don't need a new namer class for names related to tests - i.e. maybe we should just put it in SurfaceNamer.